src/components/HallOfFameButton/HallOfFameButton.js

```diff
@@ -0,0 +1,8 @@
+import * as classes from './style.module.scss';
+
+const HallOfFameButton = document.createElement('button')
```

Comment:
Those still aren't custom elements [take a look at](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements). Eventually update your branch to see code reference. If you have any questions ask away 😄